tox.ini

```diff
@@ -0,0 +1,39 @@
+[tox]
```

Comment:
@MartinBasti; Have you seen https://www.seanh.cc/2018/11/14/tox/ ?

Reply:
FWIW, osbs-integration-tests' 'tox.ini' is structured like that :arrow_up:

Reply:
so do you want me to change it into that format?